WordPress/gutenberg
# Add border controls to Cover and Featured Image blocks

<img width="1242" alt="Screenshot 2022-01-07 at 16 20 52" src="https://user-images.githubusercontent.com/846565/148573664-7d2254b4-6222-486d-aeeb-b7dd015d3940.png">

This PR makes it possible to add border radii, color, width, and style to the following blocks:

* Cover
* Featured Image

### To test Cover

* Insert a Cover block
* Add border options like color, width, radius, style
* Check that the correct values are rendered in the Editor and on the frontend
* Try adding image and video backgrounds, then check everything still renders correctly

### To test Featured Image

* Edit a post without a featured image set
* Insert the Featured Image block
* Add border options like color, width, radius, style
* Check that the featured image placeholder renders those options on the canvas
* Upload (or select from media library) a featured image
* Ensure the options still render correctly, both on the canvas and on the frontend
